harmonica/version.py

```diff
@@ -1,9 +1,11 @@
 # pylint: disable=invalid-name
 """
-Get the version number and commit hash from versioneer.
+Get the version number and commit hash from setuptools-scm.
 """
-from ._version import get_versions
+from pkg_resources import get_distribution
 
 
-full_version = get_versions()["version"]
-git_revision = get_versions()["full-revisionid"]
+# Get semantic version through setuptools-scm
+version = get_distribution("harmonica").version
+# Append a "v" before the semver version so it looks like: v0.1.0
+full_version = "v" + version
```

Comment:
Do you agree with this @leouieda?

Reply:
I'll add the `"v"` before the version number in order to keep the same style as we are using now in Fatiando.
If we decide to change it we can do it afterwards.